container/jbossweb/pom.xml

```diff
@@ -71,7 +71,7 @@
         <profile>
             <id>AS7</id>
             <activation>
-                <activeByDefault>false</activeByDefault>
+                <activeByDefault>true</activeByDefault>
```

Comment:
Ok, I see it now. Those names are screwed up, those artefacts are "jboss.web" and "org.jboss.web" each set according to the given profile. So you need to compile with:

`mvn clean install -PAS7`